src/Aspire.Hosting/Dcp/DcpVersion.cs

```diff
@@ -5,6 +5,6 @@ namespace Aspire.Hosting.Dcp;
 
 internal static class DcpVersion
 {
-    public static Version MinimumVersionInclusive = new Version(0, 1, 43);
+    public static Version MinimumVersionInclusive = new Version(0, 1, 52);
     public static Version MaximumVersionExclusive = new Version(0, 2, 0);
```

Comment:
Will setting an upper bound here become a maintenance issue?

Reply:
Depends on how good we are about SemVer, but minimum version is probably the important piece; I’m fine with removing the max version check.

Reply:
Removed the MaximumVersion check.